frontend/src/common/queries/wheresMyGene.ts

```diff
@@ -531,6 +556,8 @@ export function useFilterDimensions(version: 1 | 2 = 1): {
       );
     }
 
+    console.log(intersect);
```

Comment:
generally we want to avoid merging print statements into `main`

Reply:
+1

Reply:
Oops thank you for catching!!